content/getting-started/install.md

```diff
@@ -75,8 +111,41 @@ echo 'export PATH=$PATH:~/.jx/bin' >> ~/.bashrc
   scoop update jx
   ```
 
+### Google Cloud Platform (GCP)
+
+To run `jx` commands from the GCP Cloud Shell, install the Jenkins X
+binaries from the GitHub repository and install it via GCP shell
+commands:
+
+1.  Open the [GCP Cloud
+    Shell](https://cloud.google.com/shell/docs/starting-cloud-shell),
+    and choose your GCP project for Jenkins X.
+
+    {{% note %}}
+    It is highly recommended that you use Google Chrome browser with
+    GCP Cloud Shell, as you may experience issues using other
+    browsers.
+    {{% /note %}}
+
+2.  In GCP Cloud Shell, download the Jenkins X command-line binaries
+    latest version (where \[LATEST\_VERSION\] is the most current
+    version of Jenkins X released, which is 2.0.46):
+
+        curl -L https://github.com/jenkins-x/jx/releases/download/[LATEST_VERSION]/jx-linux-amd64.tar.gz | tar xzv
+
+3.  Move the `jx` exectutable into the executable directory with this
+    command:
+
+        sudo mv jx /usr/local/bin
```

Comment:
and there you go! :-) inconsistency with plain Linux install.

Reply:
fixed to match macOS install instructions. thanks!